index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
```

Comment:
This document uses 2 spaces for indentation throughout but the 4 tags here in the head are indented with a tab